k8-manifests/release/inventory.yaml

```diff
@@ -31,7 +31,7 @@ spec:
       containers:
       - name: inventory
         # TODO:: remove the snapshot tag once we make the first release
-        image: us-docker.pkg.dev/point-of-sale-ci/pos-images/inventory:v1-snapshot
+        image: us-docker.pkg.dev/point-of-sale-ci/pos-images/inventory:v0.0.0
```

Comment:
hmm maybe this should 0.1.0? (from https://stackoverflow.com/questions/38252708/is-0-0-1-valid-semver)

Reply:
I agree that the first version must be `0.1.0`. Currently there is no artifact for `0.0.0`. I've kept it to that so that when we trigger the first release _(after merging this PR)_, the first version will be `0.1.0`.